### DISCUSSION TITLE
merge the logs

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#787] [Ideas]
so, i'm working on the request at #763. Here is what i've done so far.
<img width="1522" alt="image" src="https://github.com/user-attachments/assets/fac3bef0-3dd1-46e4-9d0f-164be66c2ff2">
the idea is to have also a log in the monitoring page, because active connections can be very quick, and not last enough to understand what's going on. The log section can be hidden to give the list on the left the full width.
While in the logs page you have separate logs for successful and failed requests, here you would have a single list, so that all requests are strictly in time order, because it better fits the purpose of this page, that is to follow what's going on in real time.

But now that I think of it, I'm considering to merge the access+error logs as a single file, and have a single tab in the Logs page. This is what I want your opinion on, as it's a major change.
I made it separate from the beginning just because Apache does so, but it's not necessarily better, and I see that nginx for example does a single log (don't be fooled by nginx's error log, it is not for access-errors, but for non-http errors, the kind that in hfs at the moment go in the console and in the debug.log file).
Before going with this change I'd like to know if you agree or not. I'm asking everyone, but let me tag a few to get some attention @PH5HP @damienzonly @SanokKule @W-i-n-7

**Comment by SanokKule** [comment]
In my opinion, if you are going to merge logs into one, errors having a different color background would help seeing them at a glance

**Comment by W-i-n-7** [comment]
i would like to have both options

**Comment by damienzonly** [comment]
i agree with @SanokKule, i understand the reason why you separated the files and i actually prefer that way, technically speaking you're right, you don't HAVE TO separate the file, but from a UX pov you should make it easy to distinguish error logs from other ones, maybe considering a log filter if you're able to classify each log line.
regarding making a single page for monitoring + logs is fine to me, it makes sense, but i don't wanna lose the "errors" and "IPs" tabs because i use them a lot and it makes it easy for me to take action, for example, when i see unauthorized access attempts or bots randomly sending requests, and obviously every other kind of server error, but personally i care about having a dedicated/filtered section only for errors. in addition, it's good to have them in a single place so that i can have more context before, after, and anytime in between the errors occur, and be able to filter logs in a time range

**Comment by PH5HP** [comment]
Hi,

Both options would be nice if we talk about the HFS interface log section.Merging the log file and the error log file to one file seems no problem to me.

Kind regards,
Henk - PH5HP - https://fileserver.ph5hp.nl
